server/game/cards/03-WC/ObsidianForge.js

```diff
@@ -0,0 +1,33 @@
+const Card = require('../../Card.js');
+
+class ObsidianForge extends Card {
+    setupCardAbilities(ability) {
+        this.action({
+            condition: context =>
+                context.player.creaturesInPlay.length > 0,
+            target: {
+                optional: true,
+                mode: 'unlimited',
+                controller: 'self',
+                cardType: 'creature',
+                gameAction: ability.actions.sacrifice()
+            },
+            then: {
+                may: 'forge a key at +6 current cost, reduced by 1A for every destroyed creature',
```

Comment:
change A to amber and it will auto render as a symbol on the client

Reply:
resolved in latest pr